benchmark/benchmark.toml

```diff
@@ -0,0 +1,19 @@
+# TOML document for benchmark tests
+
+title = "TOML Example"
+atom = 6.626e-34
+cash = true
+chars = ["a", "b", "?", "!", "`", "-", "  d", "\t", "\\", "\"" ]
+digits = [1.0, 2.0, nan, -nan, +inf, -inf, 1.0]
+bool = [false, true, false]
+# date = [1979-05-27]
+# today = 1979-05-27T07:32:00Z
+
+[fruit]
+name = "apple"
+description = "An apple is a sweet, edible fruit produced by an apple tree (Malus pumila). Apple trees are cultivated worldwide, and are the most widely grown species in the genus Malus. The tree originated in Central Asia, where its wild ancestor, Malus sieversii, is still found today."
```

Comment:
I think here should be used multiline string syntax

Reply:
Do you mean [String]?

Reply:
@willbasky I mean multiline raw strings like this one:

```toml
lines  = '''
The first newline is
trimmed in raw strings.
   All other whitespace
   is preserved.
'''
```